portfolio/src/main/webapp/script.js

```diff
@@ -48,17 +48,32 @@ function makeCollapsible() {
   }
 }
 
-
-// Fetches a random quote from the server and adds it to the DOM.
-function getRandomQuote() {
+// Fetches adds comments to the DOM.
+function getComments() {
   fetch('/data')
     .then(response => response.json())
     .then((comments) => {
-      document.getElementById('quote-container').innerText = "";
+      const comments_container = document.getElementById('comments-container');
```

Comment:
nit:  const usually means you won't be modifying the variable, but the appendChild call below does modify it.  Interesting javascript allows for this (or const semantics might be slightly different from other languages).  I would suggest switching to `var` or `let` instead of `const`